wallet/test/Test/Pos/Util/MnemonicsSpec.hs

```diff
@@ -0,0 +1,63 @@
+module Test.Pos.Util.MnemonicsSpec (spec, Entropy(..)) where
+
+import           Universum
+
+import           Data.ByteString.Char8 (pack)
+import           Data.Set (Set)
+import           Test.Hspec (Spec, describe, it, shouldSatisfy, xit)
+import           Test.Hspec.QuickCheck (modifyMaxSuccess)
+import           Test.QuickCheck (Arbitrary (..), property)
+import           Test.QuickCheck.Gen (Gen (..), oneof, vectorOf)
+import           Test.QuickCheck.Random (mkQCGen)
+
+import           Pos.Util.BackupPhrase (BackupPhrase (..), safeKeysFromPhrase)
+import           Pos.Util.Mnemonics (defMnemonic, fromMnemonic, toMnemonic)
+import           Pos.Wallet.Web.ClientTypes.Functions (encToCId)
+import           Pos.Wallet.Web.ClientTypes.Types (CId)
+
+import qualified Data.Set as Set
+
+
+spec :: Spec
+spec = do
+    describe "Pos.Util.Mnemonics" $ do
```

Comment:
If this test is using the `hspec` autodiscovery, then this describe isn't necessary. `hspec`'s autodiscovery will take the `...Spec` module name, strip off `Spec`, and make a describe for that. So this module would get a line like:

```haskell
describe "Test.Pos.Util.Mnemonics" Test.Pos.Util.MnemonicsSpec.spec
```